image_writers_guide/sti_testing.adoc

```diff
@@ -10,64 +10,33 @@
 
-== The Goal
-
-As described in the STI documentation, STI requires 'assemble' and 'run'
-commands to be present in order to successfully execute the STI build.  You can
-also provide the 'save-artifacts' command for reusing the build artifacts and
-the 'usage' command that prints the usage to console when someone runs the
-Docker image outside the STI.
-
-The goal of testing the STI image is to make sure that all commands described
-above work properly, even if the base Docker image changed or the tooling that
-the commands are using was updated.
+The goal of testing an STI image is to make sure that all of these described commands work properly, even if the base Docker image has changed or the tooling that the commands use was updated.
 
 == Testing Requirements
+The standard location for the `test` script is [filename]#test/run#. This script is invoked by the OpenShift STI image builder and it could be a simple Bash script or a static Go binary.
 
-The standard location for the 'test' command is 'test/run'. This command is
-invoked by the OpenShift STI image builder and it could be a simple Bash script
-or a static Go binary.
-
-Since the 'test/run' command will perform the STI build, you should have STI
-binary available in your system. See the STI
-https://github.com/openshift/source-to-image/blob/master/README.md#installation[README]
-file and follow the installation instructions.
+Because the `test/run` script performs the STI build, you must have the STI binary available in your system. See the STI https://github.com/openshift/source-to-image/blob/master/README.md#installation[README] file and follow the installation instructions if required.
 
-Since the STI is putting together the application source code and Docker image,
-in order to test it, we  need a sample application source we can use during the
-test to verify the application source gets successfully converted into a Docker
-image. The sample application should be simple, but it should also exercise the
-'assemble' command.
+Because STI puts together the application source code and Docker image, to test it you need a sample application source you can use during the test to verify that the application source is successfully converted into a Docker image. The sample application should be simple, but it should also exercise the `assemble` command.
 
-== Using sti create
+== Using `sti create`
+The STI tooling comes with powerful generation tools to speed up the process of a new STI image creation. The https://github.com/openshift/source-to-image/blob/master/docs/cli.md#sti-create[`sti create` command] produces all necessary STI scripts and testing tools along with the [filename]#Makefile#. The generated [filename]#test/run# script is very simple and needs some adjusting to be useful, but it should give you a solid ground to start developing.
```

Comment:
s/a new STI image creation/creating a new STI image

s/is very simple and needs some adjusting/must be adjusted

s/it should give you a solid ground to start/it provides a good starting point to begin